tutorials/heterogeneous/syclintro.C

```diff
@@ -0,0 +1,62 @@
+/// \file
+/// \ingroup tutorial_heterogeneous
+/// \notebook -nodraw
+/// \brief Introductory SYCL examples inside ROOT’s interpreter.
```

Comment:
`sycl` support will be optional and this tutorials might be found through some web search.  Should we add a reference on how to build/obtain a version of ROOT that can run this tutorial?